integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -67,8 +66,12 @@ export class OpfResourceLoaderService extends ScriptLoader {
     return super.hasScript(src);
   }
 
-  protected handleLoadingResourceError(src?: string) {
-    return throwError(`Error while loading external ${src} resource.`);
+  protected handleLoadingResourceError(
+    src: string,
+    resolve: (value: void | PromiseLike<void>) => void
+  ) {
+    console.error(`Error while loading external ${src} resource.`);
+    resolve();
```

Comment:
Code execution was stuck within this method as resolve/reject were not called.
If a one resource fail to load we still want to keep going for the next cta script bundle.
We could also use reject() for a better distinction between fail and success.